src/lib/storage.js

```diff
@@ -49,31 +57,36 @@ export async function upload (source: Buffer | string | Readable, filename: stri
   return result
 }
 
-export function createReadStream (path: string) {
-  console.log(`reading file from ${BUCKET}/${path}`)
-  let options = {Bucket: BUCKET, Key: path}
+export async function createReadStream (path: string) {
+  let bucket = BUCKETS[await STAGE]
```

Comment:
small detail : can we have a bucket definition instead of doing BUCKETS[await STAGE] every single time we want to use it ?